docs/data/migration/migration-tree-view-v7/migration-tree-view-v7.md

```diff
@@ -126,9 +129,9 @@ This inconsistency has been solved, all the event manager now target the root of
 
 ```diff
 -<SimpleTreeView>
-+<SimpleTreeView onItemClick={handleItemClick}>
 -  <TreeItem onClick={handleItemClick}>
-+  <TreeItem >
++<SimpleTreeView onItemClick={handleItemClick}>
```

Comment:
Do you think the updated diff formulation improves readability? 🤔 
A diff above is shown line by line. 🤔 
I'd be curious to hear input from @flaviendelangle or @noraleonte

Reply:
I kind of prefer the previous one :angel:

Reply:
Reverted.